internal/config/config.go

```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/gopasspw/gopass/pkg/gitconfig"
 )
 
+const (
+	DefaultLength     = 24
```

Comment:
`DefaultLength` is very generic. gopass is mainly concered with passwords, but does also handle other data. So we should be more specific here, e.g. `DefaultPasswordLength`?